index.html

```diff
@@ -263,47 +271,58 @@ <h5 class="">Rock</h5>
     <section class="stageDesktop m-5 h-100">
       <div class="row mb-2 h-100">
         <div class="col-12 text-center">
-          <h1 class="StagesTitle">Stages </h1>
+          <h1 class="StagesTitle">Stages</h1>
         </div>
       </div>
       <div class="row stageDesktop w-75 mx-auto mt-5 mb-3">
-        <div class="col-3">
+        <div class="col-1"></div>
+        <div class="col-2">
           <div class="card">
             <img class="card-img-top" src="Images/160323-Kendrick-Lamar-827x620.jpg" alt="HipHop stage picture">
             <div class="card-body">
               <h5 class="card-title fontD">HipHop</h5>
-              <p class="card-text fontD">Stage with artist such as Kendric, Drake, J. Cole, etc...</p>
+              <a class="card-text fontD">Stage with artist such as Kendric, Drake, J. Cole, etc...</a>
             </div>
           </div>
         </div>
-        <div class="col-3">
+        <div class="col-2">
```

Comment:
- [ ] Please, consider suggestions from the previous reviewer about using the `article` tag for cards, which help in making code more readable with the use of semantic HTML.